components/Events/EventInfo.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from "react";
+import EventCard from "./EventCard";
+import styles from "./Event.module.css";
+
+const DisplayEventInfo = (props) => {
+  const [eventData, setEventData] = useState([]);
+
+  const getEventDataHandler = async () => {
+    try {
+      setEventData(props.result);
+    } catch (error) {
+      alert("fetch failed!!!");
+      console.log(error);
+      window.location = "/500";
+    }
+  };
+  // call get data handler in the useEffect hook which runs once when the component is mounted.
+  useEffect(() => {
+    getEventDataHandler();
+  }, []);
+
+  return (
+    <div>
+      <div className={styles.eventContainer}>
+        <EventCard eventData={eventData} />
```

Comment:
also the best practice to handle this error case here with the conditional rendering. https://github.com/nishi-ai/softball-register-web/pull/60/files#diff-7c97c1ad17c63f34774324965f81661cea32f533a65c39ab03576069972e4d0eR31

Reply:
is it ok like so? [86f1d6b](https://github.com/nishi-ai/softball-register-web/pull/60/commits/86f1d6b7bb9ece035e4d7a3a971a87169446cb6e)

![Screenshot 2023-02-20 at 13 43 41](https://user-images.githubusercontent.com/60405643/220112102-47e0798f-db51-4db7-8f68-fe54ae442cdb.png)

Reply:
Looks super! 🌴